inference-engine/src/multi_device/multi_device_plugin.cpp

```diff
@@ -144,11 +144,17 @@ InferenceEngine::Parameter MultiDeviceInferencePlugin::GetMetric(const std::stri
 
 ExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadExeNetworkImpl(const CNNNetwork &network,
                                                                               const std::map<std::string, std::string>& config) {
+    return DoLoadNetwork({}, network, config);
```

Comment:
can we have the same `LoadExeNetworkImpl` name? `DoXX` looks like an alien :)

Reply:
Or we can use something like this https://github.com/openvinotoolkit/openvino/pull/5545/files#diff-f2c06d6e2bc245c592a435db56895d11e749d6cb2e4a56a329528e6fb0b331beR40-R64